backend/blender_script/lens.py

```diff
@@ -362,7 +362,7 @@ def add_lens(self, context):
         #apply smooth shading
         
 
-        if self.shade_smooth:
+        if self.shade_smooth and False:
```

Comment:
What's the reason for this line?

Reply:
it would crash otherwise hehe. I think it's because at the lenses are fetched from a hardcoded index at this line which means we don't always select the correct object: https://github.com/Lenscrafters-Capstone/LensCrafters-free/blob/ca75dd47699d25fdb8f6f59ac00586a7fd252f22/backend/blender_script/main.py#L80C1-L83C36

I can try to fix it tomorrow, but I figured shading wasn't that important atm. correct me if im wrong